sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/OperationKind.java

```diff
@@ -11,6 +11,13 @@
 /** Known values for type of operation. */
 @Immutable
 public final class OperationKind extends ExpandableStringEnum<OperationKind> {
+
+    /**
+     * Creates a OperationKind object.
+     */
+    public OperationKind() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.